extensions/wikia/AnalyticsEngine/AnalyticsProviderIVW3.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+class AnalyticsProviderIVW3 implements iAnalyticsProvider {
+
+	private static $libraryUrl = 'https://script.ioam.de/iam.js';
+
+	static public function onInstantGlobalsGetVariables( array &$vars )
+	{
+		$vars[] = 'wgSitewideDisableIVW3';
+
+		return true;
+	}
+
+	function getSetupHtml( $params = array() ) {
+		return null;
+	}
+
+	function trackEvent( $event, $eventDetails = array() ) {
+		if ( !self::isEnabled() ) {
+			return '<!-- IVW3 disabled -->';
+		}
+
+		switch ($event) {
+			case AnalyticsEngine::EVENT_PAGEVIEW:
+				return $this->trackPageView();
+			default:
+				return '<!-- Unsupported event for IVW3 -->';
+		}
+	}
+
+	static public function isEnabled() {
+		global $wgSitewideDisableIVW3, $wgNoExternals;
+
+		return !$wgSitewideDisableIVW3 && !$wgNoExternals;
+	}
+
+	private function trackPageView() {
+		global $wgCityId;
+
+		$vertical = HubService::getVerticalNameForComscore( $wgCityId );
+		$url = self::$libraryUrl;
+		$iamData = json_encode( [
+				'st' => 'wikia',
+				'cp' => $vertical,
+				'sv' => 'ke',
+		] );
+
+		$ivwScriptTag = <<<CODE
+<script type="text/javascript" src="{$url}"></script>
+<script type="text/javascript">window.iom.c({$iamData}, 2);</script>
+CODE;
+		$ivwScriptTagEscaped = json_encode($ivwScriptTag);
+
+		return <<<CODE
+<!-- Begin IVW3 Tag -->
+<script type="text/javascript">
+	if (window.Wikia && window.Wikia.geo.isProperGeo(['DE', 'AT', 'CH'])) {
```

Comment:
Now I'm thinking about not an instant global but just a regular WF variable which we can use here in PHP and re-use in Mercury. But can we use it in Mercury? Like get it on first view where whole Ember is loading and Mercury asks for WF config? Can we return it with this config if it gets there before sending request to IVW? If yes, let's do it! If no, let's not waste our time and use it to think how make Mercury more flexible ;-)